File: Modul-04/SOAL 2/PRAK402-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    int MAKS, BIL;

    printf("Input: \n");
    scanf("%d", &MAKS);

    printf("Output: \n");
    for (BIL = 1; BIL <= MAKS; BIL++) {
        if (BIL % 2 != 0) {
            printf("%d ", BIL);
        }
    }
    printf("\n");
    for (BIL = MAKS; BIL >= 2; BIL--) {
        if (BIL % 2 == 0) {
            printf("%d ", BIL);
        }
    }

    return 0;
}

File: Modul-04/SOAL 2/PRAK402-2310817210009-MuhammadIbnuSina.py
MAKS = int(input("Input: \n"))
print("Output: ")

for BIL in range(1, MAKS + 1):
    if BIL % 2 != 0:
        print(BIL, end=" ")

print()

for BIL in range(MAKS, 1, -1):
    if BIL % 2 == 0:
        print(BIL, end=" ")
